### PR TITLE
basscss links to project page

### PR DESCRIPTION
**Pull request by roolo**
For a newcomer this looks like typo "Apply basscss classes" / "Apply base css classes"

### CODE REVIEW

File: www/getstarted.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h2 class="www-heading relative pb4 mb2">Step 3: Customize the page structure</h
       <section class="mb4">
         <h2 class="www-heading relative pb4 mb2">Step 4: Customize the styles</h2>
         <p>
-         Apply basscss classes or AMP Start classes to your HTML to apply built-in styles. For more fine-grained styling, you can modify/override the CSS for these classes (just remember to minify your CSS when you’re done).
+         Apply <a href="http://basscss.com">basscss</a> classes or AMP Start classes to your HTML to apply built-in styles. For more fine-grained styling, you can modify/override the CSS for these classes (just remember to minify your CSS when you’re done).
```
Comment (camelburrito) [inline www/getstarted.html]: `class="text-decoration-none"`

**Comment by camelburrito** [comment]
@roolo - Thanks for the PR - i added one comment , i can merge this when the comment has been addressed.

File: www/getstarted.html (hunk 1 above)
Comment (roolo) [reply www/getstarted.html]: @camelburrito Class in place

**Comment by camelburrito** [comment]
@roolo - Can you pelase sign the CLA so that i can get this merged?

**Comment by roolo** [comment]
RE @camelburrito — https://github.com/ampproject/ampstart/pull/626#issuecomment-337007050

Rather not. It's asking pretty private stuff for not even one full line of code.

**Comment by camelburrito** [comment]
@roolo - in that case , will that be okay if i cloned your PR and submit it? I can give you credits on the PR description.

**Comment by roolo** [comment]
RE @camelburrito -- https://github.com/ampproject/ampstart/pull/626#issuecomment-337010995

Yep, absolutely ok with that

**Pull request closed without merging**